src/std/sys.c

```diff
@@ -163,14 +163,18 @@ HL_PRIM void hl_sys_exit( int code ) {
 
 HL_PRIM double hl_sys_time() {
 #ifdef HL_WIN
+	#define EPOCH_DIFF	(134774*24*60*60.0)

```

Comment:
What is this value based on?

Reply:
it's epoch diff between Unix epoch and Windows FILETIME epoch (started January 1, 1601 (UTC))